client/lib/sentry/index.ts

```diff
@@ -0,0 +1,211 @@
+// NOTE: This file has been copied mostly verbatim from Redpop/Tumblr. The main
+// purpose is to stub basic Sentry methods so that consumers in Calypso can use
+// them. Then we can async-load Sentry for only a certain percent of requests.
+// Otherwise, we'd add a fair amount to the bundle size when we don't really need
+// it for every single request.
+import config from '@automattic/calypso-config';
+import type * as SentryApi from '@sentry/react';
+
+// Static sentry configuration. We add any dynamic values when initializing Sentry.
+const SENTRY_CONFIG: SentryApi.BrowserOptions = {
+	dsn: 'https://61275d63a504465ab315245f1a379dab@o248881.ingest.sentry.io/6313676',
```

Comment:
Do you think we should create a new Sentry project for Calypso?

Reply:
Ah, disregard this. I see it's already there :)

Reply:
This is the new project for Calypso (https://sentry.io/organizations/a8c/projects/calypso/?project=6313676)